pkg/nsx/services/securitypolicy/firewall.go

```diff
@@ -28,59 +24,17 @@ const (
 	ProjectTagCount             int = 1
 )
 
-type SecurityPolicyService struct {
+type Service struct {
```

Comment:
As commented in expand.go, please change type Service back to SecurityPolicyService

Reply:
ok